graphql_compiler/schema_transformation/rename_query.py

```diff
@@ -0,0 +1,96 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from copy import deepcopy
+
+from graphql.language import ast as ast_types
+from graphql.language.visitor import Visitor, visit
+
+from ..exceptions import GraphQLValidationError
+
+
+def rename_query(ast, renamings):
+    """Translate names of types and root vertex fields using renamings.
+
+    Besides root vertex fields (fields of the query type), no other fields will be renamed.
+
+    Args:
+        ast: Document, representing a valid query. It is assumed to have passed GraphQL's
+             builtin validation -- validate(schema, ast) in graphql/validation/validation.py --
+             in that it has the structure of a valid query, does not reference non-existent
+             types or fields, and passes type checks. The ast is not modified by this function
```

Comment:
nit: in plain English (i.e. when not referring to a Python argument), let's make sure to always capitalize AST since it's an acronym.